chain/epoch_manager/src/lib.rs

```diff
@@ -2437,6 +2437,42 @@ mod tests {
         assert_eq!(em.get_epoch_info(&EpochId(CryptoHash::default())).unwrap(), &epoch_info)
     }
 
+    #[test]
+    fn test_fishermen_unstake() {
+        let stake_amount = 1_000;
+        let fishermen_threshold = 100;
+        let validators = vec![("test1", stake_amount), ("test2", fishermen_threshold)];
+        let mut em = setup_epoch_manager(
+            validators,
+            2,
+            1,
+            1,
+            0,
+            90,
+            70,
+            fishermen_threshold,
+            default_reward_calculator(),
+        );
+        let h = hash_range(5);
+        record_block(&mut em, CryptoHash::default(), h[0], 0, vec![]);
+        // fishermen unstake
+        record_block(&mut em, h[0], h[1], 1, vec![stake("test2", 0)]);
```

Comment:
add a test where the fisherman changes stake to `1` - seems like kickout sets differ

Reply:
Fixed